CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.54.3] - 2024-07-17
+### Fixed
+- Reintroduced `ListablePropertyType` that was removed in `7.37.0`. This is used to represent all container properties
+  that can be listed. This is an internal change and should not affect users of the SDK, but might affect developers
+  that have created libraries that depend on the SDK.
```

Comment:
I think this can be removed from the changelog as it should be "internal SDK details". Only the actual types like `Date`, `DirectRelation` or `Float32` are explicitly made available to the user in e.g. `cognite/client/data_classes/data_modeling/__init__.py` and these still work the same.